config/config.go

```diff
@@ -16,6 +16,7 @@ type Config struct {
 	Labels    []string        `validate:"dive"`
 	Interval  int
 	Git       GitConfig
+	OwnerMap  map[string]string `yaml:"ownerMap"`
```

Comment:
The other yaml keys are in kebab-case, we should do the same for this one.
May we could rename it codeowner-override, or codeowner-mapping maybe ?